src/eslint-config-adeira/README.md

```diff
@@ -66,6 +66,12 @@ module.exports = {
 };
 ```
 
+Since the eslint-runner uses jest, you have to install it if you did not already do it:
+
+```
+yarn add jest --dev [--ignore-workspace-root-check]
```

Comment:
This should be added to the `"Install this package"` section just like we added there the Eslint. Jest dependency should be defined in peer deps. Could you please update it?

Reply:
Ok, but you can use the preset without jest, you cannot use the eslint-runner without jest. That is why I thought it belonged here.

Reply:
🚀

Reply:
OK, that's actually a good point. Can you revert it then? Sorry. 😅 

I was thinking that maybe we should separate the runner from the config completely.

Reply:
🤦‍♂

Reply:
✅ Thankfully, it is hard to get rid of things with git 😄

Reply:
> I was thinking that maybe we should separate the runner from the config completely.

It could be a part of monorepo-utils?

Reply:
> It could be a part of monorepo-utils?

It could be a new NPM package. The idea is to be able to use it even though you are not using our config. It's not necessarily related to it.

Reply:
Yeah, that is a good point 👍

Reply:
Yeah, that is a good point 👍

Reply:
So, is it ready to be merged?